dpypelines/pipeline/shared/transforms/sanity_check.py

```diff
@@ -1,15 +1,40 @@
 from pathlib import Path
 
-import pandas as pd
+
+def sdmx_sanity_check_v1_file_extension(sdmx_file: Path):
```

Comment:
if you're using this I think you'll need a config that specifies that you're using this. You specify the sanity checker you;'re using for a given input in the config in the "transform_inputs" field.

Reply:
Thanks.
`sdmx_sanity_check_v1(sdmx_file: Path)` is specifed in the "transform_input" field in the configuration.py.

A sanity check that the recieved sdmx file ends with `".xml"` is also included in  `sdmx_sanity_check_v1(sdmx_file: Path)`  as it is in `sdmx_sanity_check_v1_file_extension(sdmx_file: Path)`. So, I guess having the function `sdmx_sanity_check_v1_file_extension(sdmx_file: Path)` on `sanity_check.py` is probably redundant or a repetition.

Reply:
it looks redundant to me. I'd delete it if its not being called.